lib/provisioning/ProvisionDispatcher.js

```diff
@@ -346,11 +345,4 @@ class ProvisionDispatcher {
     }
 }
 
-ProvisionDispatcher.constants = {
```

Comment:
Did you move the definition of those constants elsewhere? As far as I can see they're still used.